core/src/main/scala/spire/math/Algebraic.scala

```diff
@@ -1349,7 +1349,8 @@ object Algebraic extends AlgebraicInstances {
     }
 
     @nowarn
-    def apply(expr: Algebraic.Expr): Bound = checked {
+    // TODO Restore the checked call
+    def apply(expr: Algebraic.Expr): Bound = {
```

Comment:
This `checked` call fails during th macro expansion but apparently no tests is reaching here. Maybe I should make it return a `???` to ensure eventual callers don't get tripped

Reply:
It doesn't even compile

Reply:
Turns out we need to keep the code to pass the tests though the `checked` case isn't being tested.
I think we should fix this before merging